pkgs/by-name/gp/gpu-screen-recorder-gtk/package.nix

```
@@ -19,19 +19,19 @@
   wayland,
   wrapGAppsHook3,
   wrapperDir ? "/run/wrappers/bin",
+  gitUpdater,
 }:
 
-stdenv.mkDerivation (finalAttrs: {
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293

---

Using `finalAttrs` instead of `rec` for `version` here would lead to #310373. I don't think there is a consensus on the best solution, but `rec` works fine here.